src/Tribe/Plugin_Download_Notice.php

```diff
@@ -0,0 +1,72 @@
+<?php defined( 'WPINC' ) or die;
+
+if ( ! class_exists( 'Tribe__Plugin_Download_Notice' ) ) {
+
+	/**
+	 * Shows an admin notice telling users which requisite plugins they need to download
+	 */
+	class Tribe__Plugin_Download_Notice {
```

Comment:
I love the idea of getting rid the notice class using the new stuff in 4.3. But this in intended for 4.2.7. What if I tagged this with a todo and ticketed removing it once 4.3 is released? That way we don't invest anymore time in something that's soon to go away.